content/en/service_catalog/manage_entries.md

```diff
@@ -97,8 +97,28 @@ extensions:
     customField2: customValue2
 {{< /code-block >}}
 
-## Manage service-related workflows
-[Workflow Automation][14] allows you to automate end-to-end processes across your teams. It integrates with Datadog's Service Catalog to enable dynamic and self-service workflows.
+## Automate Common Workflows
+[Workflow Automation][14] and [App Builder][24] allow you to automate end-to-end processes across your teams. They integrate with Datadog's Service Catalog to enable dynamic and self-service workflows.
+
+{< callout url="https://www.datadoghq.com/dg/datadog-app-builder/" btn_hidden="false" header="Join the Beta!">}}
+Datadog App Builder is in private beta. Complete the form to request access.
+{{< /callout >}}
+
+### Create Software Template Workflows Using Cookiecutter
+To setup software templates in Datadog, start by setting up a git repository with the template you wish to use, and then navigate to [Workflow Automation][14] to setup in Datadog. A common choice for templating is [Cookiecutter](https://www.cookiecutter.io/), an open-source project for project templates and auto-generating new projects. 
+1. **Create a workflow with below steps**:
+   - Download the template repository files
+   - Generate the new project files from the template via Cookiecutter
+   - Deploy the new project files back to the repository
+2. **Navigate to App Builder**:
+   - Create a new app for templating
+   - Create a form that collects the required parameters for your template
+   - Add a button that submits the form, triggers the workflow created in Step 1, and passes in the parameters for your template
+   - Save & publish the app
+
+  {{< img src="tracing/service_catalog/workflowtemplate.png" alt="Workflow for building software template automation" style="width:100%;" >}}
+
+  {{< img src="tracing/service_catalog/template-application.png" alt="Application for managing software templates through App Builder" style="width:100%;" >}}
```

Comment:
@dgreen15 We'll need to update these images to remove PII (like names). We have a tool called Pawparazzi to help with that. You can read more about that here: https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1985314888/Screenshots+and+Videos. I'm also happy to help if needed!